pom.xml

```diff
@@ -25,13 +25,13 @@
     <name>Kyuubi Project Parent</name>
     <version>1.1.0-SNAPSHOT</version>
     <modules>
-        <module>kyuubi-common</module>
-        <module>kyuubi-ha</module>
+        <module>dev/kyuubi-codecov</module>
         <module>externals/kyuubi-download</module>
         <module>externals/kyuubi-spark-sql-engine</module>
-        <module>kyuubi-main</module>
         <module>kyuubi-assembly</module>
-        <module>dev/kyuubi-codecov</module>
+        <module>kyuubi-common</module>
+        <module>kyuubi-ha</module>
+        <module>kyuubi-main</module>
```

Comment:
why shall we reorder these, the previous one is intentional

Reply:
Just alphabetically. What is the previous sorting logic?

Reply:
the expected build order

Reply:
Now the dependence between modules declared in specific module's `pom.xml` and managed by maven.